tensorboard/backend/event_processing/data_provider_test.py

```diff
@@ -91,6 +93,27 @@ def setUp(self):
                         name, tensor * i, step=i, description=description
                     )
 
+        logdir = os.path.join(self.logdir, "mondrian")
+        with tf.summary.create_file_writer(logdir).as_default():
+            data = [
+                ("red", (221, 28, 38), "top-right"),
+                ("blue", (1, 91, 158), "bottom-left"),
+                ("yellow", (239, 220, 111), "bottom-right"),
+            ]
+            for (name, color, description) in data:
+                image_1x1 = tf.constant([[[color]]], dtype=tf.uint8)
+                for i in xrange(1, 11):
+                    k = 6 - abs(6 - i)  # 1, .., 6, .., 2
```

Comment:
Could you add a comment like "Use a non-monotonic sequence of sample sizes to test max_length calculation" or something?  Wasn't obvious to me why the formula for k had to be this way otherwise.

Reply:
Ah, yes, sure. (Yes, that’s the reason.) Done.